layouts/partials/home.html

```diff
@@ -0,0 +1,69 @@
+                    </div>
+                </div>
+                <div class="row">
+                    <p><a class="btn btn-primary btn-lg" data-toggle="tooltip" data-placement="bottom" title="" data-original-title="Run in your browser" href="http://explorer.worldwind.earth/" role="button">Web Demo</a>
+                    <a class="btn btn-primary btn-lg" data-toggle="tooltip" data-placement="bottom" title="" data-original-title="Download and Install"  href="https://github.com/NASAWorldWind/WorldWindAndroid/releases/download/v0.6.0/worldwind-examples-release.apk" role="button">Android Demo</a>
+                    <a class="btn btn-primary btn-lg" data-toggle="tooltip" data-placement="bottom" title="" data-original-title="Launch Java WebStart"  href="https://worldwind.arc.nasa.gov/java/latest/webstart/WMSLayerManager.jnlp" role="button">Java Example</a></p>
+                </div>
+            </div>
+        </div>
+    </div>
+
+    <div class="container">
+        <h2 class="text-center">What is WorldWind?</h2>
+        <p>WorldWind is a free, open source API for a virtual globe. WorldWind allows developers to quickly and easily create interactive visualizations of 3D globe, map and geographical information. Organizations across the world use WorldWind to monitor weather patterns, visualize cities and terrain, track the movement of planes, vehicles and ships, analyze geospatial data, and educate people about the Earth.</p>
+        <p>WorldWind's source code, official releases, engineering roadmap, issue tracking and more are hosted by GitHub and managed by the NASA WorldWind development team.</p>
+        <p class="text-center"><a href="https://github.com/NASAWorldWind/"><img src="./img/github-mark-32.png">  github.com/nasaworldwind</a></p>
+    </div>
+
+    <hr>
+
+    <div class="container">
+        <h2 class="text-center">Features and Benefits</h2>
+        <div class="row">
+            <div class="col-md-4">
+                <h3 class="text-center">It's open source</h3>
+                <p>Because WorldWind is completely open source, extending the API is simple and easy to do, creating a powerful platform for giving any application the means to express, manipulate and analyze spatial data. WorldWind technology can be incorporated into Windows, Mac and Linux applications, web pages and web applications, and mobile applications as well.</p>
+            </div>
+            <div class="col-md-4">
+                <h3 class="text-center">Build what you want</h3>
+                <p>WorldWind is different from a 3D globe like Google Earth because it is not a completed application targeted at end users. Instead, it is an SDK (software development kit) that software engineers can use to build their own applications. WorldWind provides a geographic rendering engine for powering a wide range of projects, from satellite tracking systems to flight simulators.</p>
```

Comment:
Minor suggestion to remove negative context: Change "WorldWind is different from a 3D globe like Google Earth because it is not a completed application targeted at end users." to "WorldWind is different from a 3D globe like Google Earth because it is not an application targeted at specific end users."